waffle-ens/test/EnsBuilder.test.ts

```diff
@@ -35,6 +35,10 @@ describe('INTEGRATION: Deploy Ens', async () => {
     expect(await ensBuilder.ens.owner(namehash('addr.reverse'))).to.eq(ensBuilder.reverseRegistrar.address);
   });
 
+  it('Provider ENS address', async () => {
+    expect(provider.network.ensAddress).to.eq(ensBuilder.ens.address);
```

Comment:
test that `provider.resolveName(...)` and `provider.lookup(...)` work in adequate cases